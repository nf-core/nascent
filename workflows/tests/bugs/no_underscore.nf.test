nextflow_pipeline {

    name "Samplesheet with no underscores"
    script "../../../main.nf"
    tag "bug"

    // https://github.com/nf-core/nascent/issues/158
    test("Should work with samples that don't have an underscore") {
        when {
            params {
                outdir = "$outputDir"
                input = "${projectDir}/tests/config/samplesheets/no_underscore.csv"
            }
        }

        then {
            assert workflow.success
            assert snapshot(
                workflow.trace.tasks().size(),
            ).match()
        }
    }
}
