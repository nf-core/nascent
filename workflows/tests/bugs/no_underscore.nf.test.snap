{
    "Should work with samples that don't have an underscore": {
        "content": [
            58
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-08-16T22:12:17.34063593"
    }
}